package/rbac/role.yaml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
This file should be removed (see my previous comment) as RBAC permissions are handled differently in a package image